_notebooks/2020-07-28-lineage.ipynb

```diff
@@ -0,0 +1,323 @@
+{
+ "cells": [
+   {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+     "# Record metadata on Kubeflow from Notebooks\n",
+     "> Demonstration of how lineage tracking works\n",
+     "\n",
+     "- toc: true \n",
+     "- badges: true\n",
+     "- comments: true\n",
+     "- categories: [jupyter]"
+    ]
+   },
+   {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+     "# Lineage Tracking\n",
```

Comment:
Maybe remove the bullet list from this paragraph?

The intent of this ...
We are going to:
1. ...
2. ...
3. ...